assets/application.json

```diff
@@ -216,22 +216,22 @@
               "title": "Instruçoes de entrega",
               "description": "Insira informações adicionais para retirada ou entrega do pedido"
             },
-            "all_product_ids": {
-              "type": "boolean",
-              "title": "Ativar frete grátis para todos itens da lista",
-              "description": "Será aplicado frete grátis se apenas estiver os produtos especificados no carrinho",
-              "default": false
-            },
             "product_ids": {
-              "title": "Produtos com frete grátis",
+              "title": "Lista de produtos para frete grátis",
               "description": "Será aplicado frete grátis se algum dos produtos especificados estiver no carrinho",
```

Comment:
https://github.com/ecomplus/app-custom-shipping/pull/10#discussion_r1123927186
https://github.com/ecomplus/app-custom-shipping/pull/10#discussion_r1123927275

Reply:
De novo, não tem essa de frete grátis aqui.
Na config da regra tem um field pra valor, se ele for 0 o frete é grátis, se não ele não é grátis, sem ambiguidade.

Reply:
Da mesma forma por exemplo em regras de desconto, não é _Lista de clientes para 100% de desconto_, é _Lista de clientes selecionados_ , o valor do desconto é outro campo da regra